tests/test_special_orthogonal2.py

```diff
@@ -0,0 +1,429 @@
+"""Unit tests for special orthogonal group SO(3)."""
+
+import warnings
+
+import tests.helper as helper
+
+import geomstats.backend as gs
+import geomstats.tests
+from geomstats import algebra_utils
+from geomstats.geometry.special_orthogonal import SpecialOrthogonal
+
+
+EPSILON = 1e-5
+ATOL = 1e-5
+
+
+class TestSpecialOrthogonal2(geomstats.tests.TestCase):
+    def setUp(self):
+        warnings.simplefilter('ignore', category=ImportWarning)
+        warnings.simplefilter('ignore', category=UserWarning)
+
+        gs.random.seed(1234)
+
+        n_seq = [2]
```

Comment:
No need for n_seq since n=2 here

Reply:
True, I lazily adapted that from the SO(3) case